xml/System.Diagnostics/Process.xml

```diff
@@ -3870,13 +3870,12 @@ p.WaitForExit();
   
  ]]></format>
         </remarks>
-        <exception cref="T:System.InvalidOperationException">No file name was specified in the <see cref="T:System.Diagnostics.Process" /> component's <see cref="P:System.Diagnostics.Process.StartInfo" />.  
-  
- -or-  
-  
+        <exception cref="T:System.InvalidOperationException">No file name was specified in the <see cref="T:System.Diagnostics.Process" /> component's <see cref="P:System.Diagnostics.Process.StartInfo" />.
+ -or-
  The <see cref="P:System.Diagnostics.ProcessStartInfo.UseShellExecute" /> member of the <see cref="P:System.Diagnostics.Process.StartInfo" /> property is <see langword="true" /> while <see cref="P:System.Diagnostics.ProcessStartInfo.RedirectStandardInput" />, <see cref="P:System.Diagnostics.ProcessStartInfo.RedirectStandardOutput" />, or <see cref="P:System.Diagnostics.ProcessStartInfo.RedirectStandardError" /> is <see langword="true" />.</exception>
         <exception cref="T:System.ComponentModel.Win32Exception">There was an error in opening the associated file.</exception>
         <exception cref="T:System.ObjectDisposedException">The process object has already been disposed.</exception>
+        <exception cref="T:System.PlatformNotSupportedException">Method not supported on Nano Server machines (.NET Core only).</exception>
```

Comment:
I think this might happen on Windows Server Core and possibly Windows IoT as well. Also on Linux distros without certain shell compnents. So you might want a more general statement like `Method not supported on machines that do not have a shell (.NET Core only)` or something.

Reply:
Or `... do not have suitable shell support ..` or something

Reply:
Ok. I'll add Nano Server as an example then.